diff --git a/types/index.d.ts b/types/index.d.ts
index 639234f7e9181546ea17b8f98a198f4c5a66106d..7c923425a6426dc0a9b1e6111c7efa927556d32c 100644
--- a/types/index.d.ts
+++ b/types/index.d.ts
@@ -20,9 +20,9 @@ import type {
   PoolClusterConfig,
   SqlImportOptions,
   ConnectionInfo,
-  UserConnectionConfig
-} from './share';
+  UserConnectionConfig
+} from './share.js';
 
-export * from './share';
+export * from './share.js';
 
 export const version: string;
