diff --git a/binding.gyp b/binding.gyp
index 0250be81f9ade3c0e0eec8964c4e1f3b141760c4..9830c57443638805d4ca77a6a8155bfa03036f09 100644
--- a/binding.gyp
+++ b/binding.gyp
@@ -21,7 +21,7 @@
         "-O2"
       ],
       "msvs_configuration_attributes": {
-        "SpectreMitigation": "Spectre"
+        "SpectreMitigation": "false"
       },
       "msvs_settings": {
         "VCCLCompilerTool": {
diff --git a/deps/sqlite3.gyp b/deps/sqlite3.gyp
index 4b8400192111f4094896d2d4a6e0d9b21afa5502..b25f7c6f2ff106a2bfd0f461231a04807da86de5 100644
--- a/deps/sqlite3.gyp
+++ b/deps/sqlite3.gyp
@@ -29,7 +29,7 @@
       }
     },
     'msvs_configuration_attributes': {
-      'SpectreMitigation': 'Spectre'
+      'SpectreMitigation': 'false'
     },
     'msvs_settings': {
       'VCCLCompilerTool': {
